src/install.rst

```diff
@@ -184,6 +184,23 @@ Set up a Nextstrain runtime
             .. include:: snippets/nextstrain-setup-conda.rst
 
 
+   .. group-tab:: Singularity
+
+      Singularity is a container system freely-available for Linux platforms. It is commonly available on institutional HPC systems as an alternative to Docker, which is often not supported on such systems.
+
+      1. Check that Singularity is available in your environment.
+
+         .. code-block:: bash
+
+            singularity --version
```

Comment:
The Docker section in this doc describes how to install/configure Docker itself on different OS/platforms. We're assuming that's already done for Singularity, I guess? and that it doesn't differ per OS/platform?

I think I'd be more inclined to replicate the second-level OS/platforms tabs, and guide folks to using Singularity on Windows and macOS where using it requires a VM (WSL2 or otherwise). This is what we do for the Conda runtime, after all.

Reply:
Or, if we want to assume Singularity is only being used in places where an administrator has installed and configured it, that we note as much and suggest that it might need enablement by the HPC's modules system (e.g. `modules load singularity` or something).

Reply:
It's easier for us to assume Singularity is used as a Docker replacement for HPC systems. That seems to be where most of the users will come from, at least for now. I've [updated](https://github.com/nextstrain/docs.nextstrain.org/compare/cb4e30fa1d26fbd8761622785db5dc20a91fdc64..0961b9eb3564f7f776c1791c8a27e3e5e5f29567) wording to make that explicit.

Reply:
This is great, thanks.